package.json

```diff
@@ -52,6 +51,7 @@
     "babel-core": "^5.8.14",
     "babel-eslint": "^4.0.5",
     "babel-loader": "^5.3.2",
+    "babel-runtime": "^5.8.12",
```

Comment:
This should not have been included in `packages.dependencies`. I'm a bit surprised that I didn't notice it earlier.